### PR TITLE
Fix running with --web

### LINKED ISSUE #25: Search doesn't work with --web flag, without it, or on the website, and I'm freaking out
* [x] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

### Command

```
/usr/local/bin/pod search PeerKit --web
```

### Report

* What did you do?
I searched a Pod, as I wasn't sure if it was something that was its' own spec or not.

* What did you expect to happen?
I haven't used CLI search before, so I don't know; probably something searchy.

* What happened instead?
I got this. Additionally, this doesn't work on the website either.

example

### Stack

```
   CocoaPods : 1.5.0
        Ruby : ruby 2.3.3p222 (2016-11-21 revision 56859) [universal.x86_64-darwin17]
    RubyGems : 2.5.2
        Host : Mac OS X 10.13.4 (17E199)
       Xcode : 9.3 (9E145)
         Git : git version 2.15.1
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib
Repositories : KittenCommonModules - https://github.com/coffeeboo/KittenCommonModules @ f50c046a44729c16740891d806782df52db46779
               master - https://github.com/CocoaPods/Specs.git @ dc738293f3d7341050af7736385d5c94fa39cfb2
```

### Plugins

```
claide-plugins        : 0.9.2
cocoapods-deintegrate : 1.0.2
cocoapods-playgrounds : 1.2.2
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.3.0
cocoapods-try         : 1.1.0
```

### Error

```
NoMethodError - undefined method `open!' for #<Pod::Command::Search:0x007fb11023f078>
Did you mean?  open
/Library/Ruby/Gems/2.3.0/gems/cocoapods-search-1.0.0/lib/cocoapods-search/command/search.rb:79:in `web_search'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-search-1.0.0/lib/cocoapods-search/command/search.rb:61:in `run'
/Library/Ruby/Gems/2.3.0/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.0/lib/cocoapods/command.rb:52:in `run'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.0/bin/pod:55:in `<top (required)>'
/usr/local/bin/pod:22:in `load'
/usr/local/bin/pod:22:in `<main>'
```

### Other issues
[“pod search --web «search text»” Command Pukes](https://github.com/CocoaPods/CocoaPods/issues/7702)

### PR DESCRIPTION
**Pull request by segiddins**
Fixes #25

### CODE REVIEW

**Review by dnkoutso — APPROVED** [review]

**Comment by paulb777** [comment]
Can this go out with the next 1.6.0 beta?

**Comment by dnkoutso** [comment]
Yes as long as it gets a green build :)

File: lib/cocoapods-search/command/search.rb [hunk 1]
```diff
@@ -76,7 +76,7 @@ def web_search
         query_parameter = queries.compact.flatten.join(' ')
         url = "https://cocoapods.org/?q=#{CGI.escape(query_parameter).gsub('+', '%20')}"
         UI.puts("Opening #{url}")
-        open!(url)
+	Executable.execute_command(:open, [url])
```
Comment (dnkoutso) [inline lib/cocoapods-search/command/search.rb]: this does not appear indented properly

**Comment by dnkoutso** [comment]
@paulb777 merged but the option is broken since the url generated is not parsed by cocoapods.org and therefore you see no search results.

But in terms of this project the crash is fixed.

**Comment by paulb777** [comment]
Maybe the `--web` option should be removed in favor of only supporting terminal output?

**Pull request merged**